site/public/css/server.css

```diff
@@ -1270,6 +1270,7 @@ div.full_height {
     border-radius: 0.3em;
     margin: 0.3em 0.1em 0.3em 0.1em;
     padding: 0.2em;
+    width:400px;
```

Comment:
Can we simply just modify the width on line 1263? Also I didn't experience this issue, are you zoomed in?

Reply:
This issue is only visible on firefox browser